lib/routes/index.js

```diff
@@ -88,3 +96,30 @@ module.exports = function routing(app, options, callback) {
     done();
   }, callback);
 };
+
+/**
+  * @swagger
+  * /api/packages/{pkg}:
```

Comment:
Can we define these closer to the actual routes they're for (e.g. https://github.com/godaddy/warehouse.ai-ui/blob/master/lib/routes/index.js#L54 & https://github.com/godaddy/warehouse.ai-ui/blob/master/lib/middlewares/index.js#L10)?

Reply:
meaning split this and move part of it up by the route it's for, and move the other into the other file.